content/_components/shopify-admin-v2/index.md

```diff
@@ -529,6 +521,6 @@ There is no configuration fields in this action.
 ## Known limitations
 
 * Look at [Shopify API rate limits](https://shopify.dev/api/usage/rate-limits), specially [GraphQL Admin API rate limits](https://shopify.dev/api/usage/rate-limits#graphql-admin-api-rate-limits)
-* If the component reaches API rate limit it will retry request after waiting until queue will be restored up to 10 times: for example - query cost `500` points, currently available only `100` points, restore rate `50` points/second, component will wait `8` seconds until available points will be restored and try again to get data
+* If the component reaches API rate limit it will retry request after waiting until queue will be restored up to 10 times: for example - query cost `500` points, currently available only `100` points, restore rate `50` points/second, component will wait `8` seconds until available points will be restored and try again to get data.
 
-  Be carefully with several flows running at the same time, each of then can affect on total available points, if component won't be able to get data after 10 retries, then error `"Throttled"` will be thrown
+  Be carefully with several flows running at the same time, each of then can affect on total available points, if component won't be able to get data after 10 retries, then error `"Throttled"` will be thrown.
```

Comment:
Be careful with several flows running at the same time, each of them can affect on total available points, if the component won't be able to get data after 10 retries, then the error `"Throttled"` will be thrown.